### PR TITLE
Translate 2019-08-28 news (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_cn/news/_posts/2019-08-28-ruby-2-4-7-released.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+layout: news_post
+title: "Ruby 2.4.7 已发布"
+author: "usa"
+translator: Delton Ding
+date: 2019-08-28 09:00:00 +0000
+lang: zh_cn
+---
+
+Ruby 2.4.6 已发布。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2019-08-28-ruby-2-4-7-released.md]:
```suggestion
Ruby 2.4.7 已发布。
```

File: zh_cn/news/_posts/2019-08-28-ruby-2-6-4-released.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+layout: news_post
+title: "Ruby 2.6.4 已发布"
+author: "nagachika"
+translator: Vincent Lin
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2019-08-28-ruby-2-6-4-released.md]:
```suggestion
translator: Delton Ding
```

File: zh_cn/news/_posts/2019-08-28-ruby-2-6-4-released.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+date: 2019-08-28 09:00:00 +0000
+lang: zh_tw
+---
+
+Ruby 2.6.4 已发布。
+
+此次发布修复了数个安全漏洞。请检查以下主题以了解详情。
+
+* [RDoc 中多个 jQeury 安全性缺陷](/zh_tw/news/2019/08/28/multiple-jquery-vulnerabilities-in-rdoc/)
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2019-08-28-ruby-2-6-4-released.md]:
```suggestion
* [RDoc 中多个 jQeury 安全性缺陷](/zh_cn/news/2019/08/28/multiple-jquery-vulnerabilities-in-rdoc/)
```

**Comment by dsh0416** [comment]
@JuanitoFatas Thank you for your reviewing. The problems mentioned have fixed.

**Pull request merged**